engine/transcoder/ffmpeg.go

```diff
@@ -16,6 +16,11 @@ type Transcoder interface {
 }
 
 func New() Transcoder {
+	path, err := exec.LookPath("ffmpeg")
+	if err != nil {
+		log.Error("Unable to find ffmpeg: %s", err)
```

Comment:
You don't to use formatting strings like this. In ND's log package, just do `log.Error("Unable to find ffmpeg", err)` and it will add do the Right Thing:tm: